### PR TITLE
get RT jobs working on new special node pool

### PR DESCRIPTION
**Pull request by atvaccaro**
### Major changes:
1. Converts all 3 RT _parsing_ jobs to pod operators running on the new special node pool in the apps cluster
2. Splits off the RT parsing code as a standalone app/image, necessary for the above
3. Adds github workflow for publishing that image

### Fixes
1. Uses service account auth for RT _validation_ job (already a pod operator)

Successful service alerts:
```
[2022-03-31 20:34:52,430] {pod_launcher.py:198} INFO - Event: parse-rt-service-alerts.7dacadb2d80240c48e30b077f758dcb7 had an event of type Succeeded
[2022-03-31 20:34:52,430] {pod_launcher.py:311} INFO - Event with job id parse-rt-service-alerts.7dacadb2d80240c48e30b077f758dcb7 Succeeded
[2022-03-31 20:34:52,638] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=rt_loader, task_id=parse_rt_service_alerts, execution_date=20220329T000000, start_date=20220330T204252, end_date=20220331T203452
```

### CODE REVIEW

**Review by lauriemerrell — COMMENTED** [review]
**question**: Should this PR delete the `FlattenedJSONtoWarehouse` operator?

**Comment by atvaccaro** [comment]
> **question**: Should this PR delete the `FlattenedJSONtoWarehouse` operator?

That's a good call, will do.

File: .github/workflows/build-gtfs-rt-parser.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: Docker image
+
+on:
+  push:
+    branches:
+      - 'master'
```
Comment (evansiroky) [inline .github/workflows/build-gtfs-rt-parser.yml:6]: @atvaccaro Should this be `main`?

Comment (evansiroky) [reply .github/workflows/build-gtfs-rt-parser.yml:6]: I guess https://github.com/cal-itp/data-infra/issues/1304 refers to this.

Comment (atvaccaro) [reply .github/workflows/build-gtfs-rt-parser.yml:6]: oops yes, thanks; resolving in PR

**Pull request merged**